dll/win32/tapiui/lang/pl-PL.rc

```diff
@@ -258,24 +259,24 @@ CAPTION "Zaawansowane"
 FONT 8, "MS Shell Dlg"
 BEGIN
     ICON 201, 2000, 7, 7, 20, 20
-    LTEXT "Następujący providerzy usług telefonicznych są zainstalowani na tym komputerze:", -1, 40, 7, 205, 8
-    LTEXT "&Providerzy:", -1, 7, 35, 238, 8
+    LTEXT "Następujący dostawcy usług telefonicznych są zainstalowani na tym komputerze:", -1, 40, 7, 205, 21
+    LTEXT "&Dostawcy:", -1, 7, 35, 238, 8
     LISTBOX 1033, 7, 46, 238, 147, LBS_STANDARD | LBS_NOINTEGRALHEIGHT | WS_TABSTOP
     PUSHBUTTON "&Dodaj...", 1005, 87, 197, 50, 14
     PUSHBUTTON "&Usuń....", 1024, 141, 197, 50, 14
     PUSHBUTTON "&Konfiguruj", 1031, 195, 197, 50, 14
 END
 
-114 DIALOGEX 20, 15, 252, 127
+114 DIALOGEX 20, 15, 252, 134
 STYLE DS_SHELLFONT | DS_MODALFRAME | DS_CONTEXTHELP | WS_POPUPWINDOW | WS_VISIBLE | WS_CAPTION
-CAPTION "Dodaj Providera"
+CAPTION "Dodawanie dostawcy"
 FONT 8, "MS Shell Dlg"
 BEGIN
-    LTEXT "Wybierz providera, którego chciałbyś zainstalować, z listy poniżej, i naciśnij Dodaj.", 2000, 7, 7, 241, 8
-    LTEXT "&Providerzy usług telefonicznych:", -1, 7, 23, 144, 10
-    LISTBOX 1055, 7, 34, 238, 74, LBS_STANDARD | WS_TABSTOP
-    PUSHBUTTON "&Dodaj", 1005, 141, 106, 50, 14
-    DEFPUSHBUTTON "Anuluj", 2, 195, 106, 50, 14
+    LTEXT "Wybierz dostawcę, którego chcesz zainstalować, z listy poniżej, i naciśnij Dodaj.", 2000, 7, 7, 241, 17
+    LTEXT "&Dostawcy usług telefonicznych:", -1, 7, 30, 144, 10
+    LISTBOX 1055, 7, 41, 238, 74, LBS_STANDARD | WS_TABSTOP
```

Comment:
Please revert stylistic Y-axis changes, they should be done separately for all languages.

See en-US and other translations: https://github.com/reactos/reactos/blob/7fe7f5e4c107451c4485cc53f140ba5e93612b63/dll/win32/tapiui/lang/en-US.rc#L267

Reply:
if i revert these changes, text is unreadable

Reply:
Alright, this one looks ok.